persistence/src/main/java/edu/unc/lib/dl/persist/services/edit/EditTitleService.java

```diff
@@ -74,25 +79,33 @@ public void editTitle(AgentPrincipals agent, PID pid, String title) throws Excep
             ContentObject obj = (ContentObject) repoObjLoader.getRepositoryObject(pid);
             BinaryObject mods = obj.getDescription();
 
-            String username = agent.getUsername();
-
-            String newMods;
+            Document newMods;
             InputStream modsStream;
-            try {
+            if (mods != null) {
                 modsStream = mods.getBinaryStream();
-            } catch (NullPointerException e) {
+            } else {
                 modsStream = null;
             }
 
             if (modsStream != null) {
-                String modsString = IOUtils.toString(modsStream, StandardCharsets.UTF_8);
+                String modsString;
```

Comment:
You don't need to pass the mods through a String before deserializing it into a document, `SAXBuilder` can take the original InputStream directly.